doc/source/examples/JobAnalysis/job_analysis.ipynb.rst

```diff
@@ -124,94 +124,102 @@ Jobs can be loaded by passing job objects directly, or loading from a path.
+/path/plams/examples/JobAnalysis/plams_workdir.003/dftb.003 dftb.003 True  True  None
+/path/plams/examples/JobAnalysis/plams_workdir.003/adf.005  adf.005  True  True  None
+/path/plams/examples/JobAnalysis/plams_workdir.003/adf.006  adf.006  True  True  None
+/path/plams/examples/JobAnalysis/plams_workdir.003/dftb.004 dftb.004 True  True  None
+/path/plams/examples/JobAnalysis/plams_workdir.003/adf.007  adf.007  True  True  None
+/path/plams/examples/JobAnalysis/plams_workdir.003/adf.008  adf.008  True  True  None
+/path/plams/examples/JobAnalysis/plams_workdir.003/dftb.005 dftb.005 True  True  None
+/path/plams/examples/JobAnalysis/plams_workdir.003/adf.009  adf.009  True  True  None
+/path/plams/examples/JobAnalysis/plams_workdir.003/adf.010  adf.010  True  True  None
+/path/plams/examples/JobAnalysis/plams_workdir.003/dftb.006 dftb.006 True  True  None
+/path/plams/examples/JobAnalysis/plams_workdir.003/adf.011  adf.011  True  True  None
+/path/plams/examples/JobAnalysis/plams_workdir.003/adf.012  adf.012  True  True  None
+/path/plams/examples/JobAnalysis/plams_workdir.003/dftb.007 dftb.007 True  True  None
+/path/plams/examples/JobAnalysis/plams_workdir.003/adf.013  adf.013  True  True  None
+/path/plams/examples/JobAnalysis/plams_workdir.003/adf.014  adf.014  True  True  None
+/path/plams/examples/JobAnalysis/plams_workdir.003/dftb.008 dftb.008 True  True  None
+/path/plams/examples/JobAnalysis/plams_workdir.003/adf.015  adf.015  True  True  None
+/path/plams/examples/JobAnalysis/plams_workdir.003/adf.016  adf.016  True  True  None
+/path/plams/examples/JobAnalysis/plams_workdir.003/neb      neb      False False NEB optimization did NOT converge
+/path/plams/examples/JobAnalysis/plams_workdir.003/neb.002  neb.002  True  True  None
+=========================================================== ======== ===== ===== =================================
 
 Adding and Removing Fields
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-A range of common standard fields can be added with the ``add_standard_field(s)`` methods. Custom fields can also be added with the ``add_field`` method, by defining a field key, value accessor and optional arguments like display name and value formatting. Fields can be removed by calling ``remove_field`` with the corresponding field key.
+A range of common standard fields can be added with the ``add_standard_field(s)`` methods. In addition, fields deriving from the job settings can be added with the ``add_settings_input_fields`` method, and fields from the output rkfs with the ``add_rkf_field`` method. Custom fields can also be added with the ``add_field`` method, by defining a field key, value accessor and optional arguments like display name and value formatting.
+
+Fields can be removed by calling ``remove_field`` with the corresponding field key.
 
 .. code:: ipython3
 
-   (
+   ja = (
        ja.remove_field("Path")
        .add_standard_fields(["Formula", "Smiles", "CPUTime", "SysTime"])
+       .add_rkf_field("General", "engine")
```

Comment:
Would prefer this in a seperate PR because I'm looking for completely different things in the diffs for the new method compared to changing the behaviour of the class.

Reply:
OK noted, next time I will split them, will leave here for now